packages/inventory/src/containers/ProfileView/index.tsx

```diff
@@ -0,0 +1,253 @@
+  LAT_LONG_LABEL,
+  COMMUNE_LABEL,
+  SERVICE_POINT_ID_LABEL,
+  BACK_TO_SERVICE_POINT_LIST,
+} from '../../lang';
+import { TableData } from './utils';
+import '../../index.css';
+import {
+  fetchInventories,
+  getInventoriesArray,
+  inventoryReducer,
+  inventoryReducerName,
+} from '../../ducks/inventory';
+import { inventory1, inventory2 } from '../../constants';
+import { getNodePath } from './utils';
+/** make sure locations and hierarchy reducer is registered */
+reducerRegistry.register(hierarchyReducerName, hierarchyReducer);
+reducerRegistry.register(locationUnitsReducerName, locationUnitsReducer);
+reducerRegistry.register(inventoryReducerName, inventoryReducer);
+
+const locationsBySearchSelector = getLocationsByNameAndId();
+const structuresSelector = getLocationsBySearch();
+const treesSelector = getTreesByIds();
+
+/** props for the ServicePointProfile view */
+interface ServicePointsListProps extends CommonProps {
+  LocationsByGeoLevel: TreeNode[];
+  columns: ColumnsType<TableData>;
+  fetchInventories: typeof fetchInventories;
+  geoLevel: number;
+  service: typeof OpenSRPService;
+}
+
+export interface Props {
+  opensrpBaseURL: string;
+}
+
+const defaultProps = {
+  ...defaultCommonProps,
+  LocationsByGeoLevel: [],
+  columns: columns,
+  fetchInventories,
+  geoLevel: 0,
+  opensrpBaseURL: '',
+  service: OpenSRPService,
+};
+
+type ServicePointsListTypes = ServicePointsListProps & RouteComponentProps & Props;
+
+export interface GeographicLocationInterface {
+  geographicLevel?: number;
+  label?: number;
+}
+
+export const findPath = (nodePath: any[], geoLevel: number) => {
+  return nodePath.find((x: GeographicLocationInterface) => x.geographicLevel === geoLevel);
+};
+
+interface DefaultGeographyItemProp {
+  label: string;
+  value?: string | number | string[] | number[];
+}
+
+/** component that renders Geography Items
+ *
+ * @param props - the component props
+ */
+export const GeographyItem = (props: DefaultGeographyItemProp) => {
+  const { label, value } = props;
+  return (
+    <Col md={24} className="geography-item">
+      <p className="item">
+        {label}: {value}
+      </p>
+    </Col>
+  );
+};
+
+/** component that renders service point list
+ *
+ * @param props - the component props
+ */
+const ServicePointProfile = (props: ServicePointsListTypes) => {
+  const { columns, opensrpBaseURL, geoLevel, service } = props;
+  const { broken, errorMessage, handleBrokenPage } = useHandleBrokenPage();
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const dispatch = useDispatch();
+  const location = useLocation();
+  const loc = location.pathname.split('/');
+  const spId = loc.pop() as string;
+
+  const inventoriesArray = useSelector((state) => getInventoriesArray(state));
+  const locationUnitById = useSelector((state) => getLocationUnitById(state, spId));
+  const [locationNodeById] = useSelector((state) =>
+    locationsBySearchSelector(state, { searchQuery: spId, geoLevel })
```

Comment:
what does this 2 represent, which one is the rootLocations, should we remove the other, is it correct to assume that we can ever only have one root jurisdiction?